engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -45,7 +45,7 @@
     private WorldProvider world;
     
     @In
-    private Config config;
+    private Config  config;
```

Comment:
extra spae

Reply:
Ah, didn't noticed this. Thanks for pointing out, I'll edit and revert with a new commit.